experimenter/evaluate_pipeline_new.py

```diff
@@ -0,0 +1,199 @@
+import itertools as it
+import json
+import yaml
+import os
+import parser
+
+from typing import Any, List, Tuple
+from uuid import UUID
+from experimenter import config, utils
+
+from d3m import cli as d3m_cli
+from d3m.contrib.pipelines import K_FOLD_TABULAR_SPLIT_PIPELINE_PATH as k_fold_split_path
+from d3m.contrib.pipelines import FIXED_SPLIT_TABULAR_SPLIT_PIPELINE_PATH as fixed_split_path
+from experimenter.databases.d3m_mtl import D3MMtLDB
+
+
+def save_pipeline_run_to_d3m_db(pipeline_run_path: str):
```

Comment:
Would it make sense to move this function to the `d3m_mtl.py` file?

Reply:
Maybe, it is using a function that is in the class on the d3m_mtl.py file. You want the class interface on the same file as the class itself? I think it could be good to have the save pipeline method except the path and load and save the file within it. And then calling and saving would only be two lines and we can put that in the evaluate function.

Reply:
Its ok to have more than just the class definition in a file. Files are the mechanism for creating modules. It seems fine to have helper functions in the same module as a class that they wrap.